doc/index.rst

```diff
@@ -2,15 +2,28 @@ xarray: N-D labeled arrays and datasets in Python
 =================================================
 
 **xarray** (formerly **xray**) is an open source project and Python package
-that aims to bring the labeled data power of pandas_ to the physical sciences,
-by providing N-dimensional variants of the core pandas data structures.
-
-Our goal is to provide a pandas-like and pandas-compatible toolkit for
-analytics on multi-dimensional arrays, rather than the tabular data for which
-pandas excels. Our approach adopts the `Common Data Model`_ for self-
-describing scientific data in widespread use in the Earth sciences:
-``xarray.Dataset`` is an in-memory representation of a netCDF file.
-
+that makes working with labelled multi-dimensional arrays simple,
+efficient, and fun!
+
+Multi-dimensional (a.k.a. N-dimensional, ND) arrays (somtimes called "tensors")
+are an essential part of computational science.
+They are encountered in a wide range of fields, including physics, astronomy,
+geoscience, bioinformatics, engineering, finance, and deep learning.
+In python, numpy_ provides the fundamental data structure and API for
+working with raw ND arrays.
+However, real-world datasets are usually more than just raw numbers;
+they have "labels" which encode information about how the array values map
+to locations in space, time, etc.
+By adopting the the `Common Data Model`_ for self-describing scientific data,
+xarray is capable of understanding these labels and using them to provide a
+more intuitive, more concise, and less error-prone experience for working with
+ND arrays.
+Xarray also provides a large and growing library of functions for advanced
+analytics and visualization with these data structures.
+Xarray was inspired by and borrows heavily from pandas_, a highly popular data
+analysis package focused on labelled tabular data.
```

Comment:
It would be nice to still see the words "netCDF" somewhere (or maybe that's implicit in our mentioning of the "Common Data Model"?).

Roughly speaking we have three audiences here:
- NumPy users who want labels
- pandas users who want to work with higher-dimensional data
- netCDF users who want good in-memory data-structures

Reply:
I removed it in response to @alexamici's comments. But in retrospect I agree that it belongs there. (I personally had never heard of CDM before xarray.)

Reply:
I would prioritize mentioning netCDF over the CDM and maybe drop CDM entirely from the brief intro. I don't think many people know what the "common data model" refers to, and worse it seems to be a heavily overloaded term, even in technical contexts (e.g., the top hit from Google is something unrelated from Microsoft).

Reply:
> 
> Roughly speaking we have three audiences here:
> 
>     * NumPy users who want labels
> 
>     * pandas users who want to work with higher-dimensional data
> 
>     * netCDF users who want good in-memory data-structures

This seems key enough that I might even put this somewhere in the docs?

and 

`* pandas users who want to work with higher-dimensional data`
-> 
` * pandas users who want to work with higher-dimensional data and an explicit, production-capable API`

Reply:
This might be good stuff to add to the “Why xarray” page.